edx_name_affirmation/api.py

```diff
@@ -212,3 +214,41 @@ def update_is_verified_status(
         )
     )
     log.info(log_msg)
+
+
+def update_or_create_verified_name_config(user, use_verified_name_for_certs=None):
+    """
+    Update or create verified name configuration for the given user.
+    Arguments:
+        * `user` (User object)
+        * `use_verified_name_for_certs` (bool): If True, certificates will prioritize
+          the user's verified name over their profile name.
+    """
+    fields = {'user': user}
+    if use_verified_name_for_certs is not None:
```

Comment:
the None path is not tested below

I'm still getting up to speed again on how python & django work but why would we even allow you to call this without that arg provided? Then you can drop the whole path.

Reply:
I do test for the case where the arg is none here: https://github.com/edx/edx-name-affirmation/blob/9409df750dae992157e377cc9dbe7a6196a377fb/edx_name_affirmation/tests/test_api.py#L272

In the case that a user does not yet have a verified name (or they haven't selected a preference) we probably would call the function without the `use_verified_name_for_certs` arg. That way we can allow rely on the default value set in the model instead of having to supply an argument in each place that the function is called.

Reply:
👍 I missed that test

also thanks for explaining the use case